### PR TITLE
Local settings for custom manager paths

### PR DESCRIPTION
**Pull request by mishanthrop**
Work In Progress, but some code review would be useful.
Prehistory: https://github.com/baoweb/modx-queeg/issues/8

image

### CODE REVIEW

**Comment by bartholomej** [comment]
@mishantrop Awesome work, Mikhail! 🎉 I will take a look tomorrow.
I just noticed one issue now: plural is "paths", not "pathes" (also name of variables) ;)

File: src/background.js [hunk 1]
```diff
@@ -1,77 +1,120 @@
...
-            var tooltip = [];
-            for(var key in message) {
-                tooltip.push(key + ": " + message[key]);
-            }
-
-            chrome.browserAction.setTitle({
-                title: tooltip.join('\n'),
-                tabId: sender.tab.id
-            });
+  chrome.storage.sync.get('pathes', function(result) {
```
Comment (bartholomej) [inline src/background.js]: Plural is "paths", not "pathes" ;)

Comment (mishanthrop) [reply src/background.js]: Oops, I will fix it, thanks.
It is a first iteration (coding night), later I will refactor.

**Comment by mishanthrop** [comment]
So, I tested the extension. It seems to work for general scenarios.

**Comment by mishanthrop** [comment]
Code is ready to merge?

**Comment by bartholomej** [comment]
@mishantrop I just tried and it works and looks great! Refactor is nice as well. Good job :) I am just merging your PR 🎉Ready to release :)

**Comment by bartholomej** [comment]
@mishantrop Released in Chrome webstore. You can update to version 2.0.0. Thanks. Great work ;) https://chrome.google.com/webstore/detail/modx-queeg/pchmcecidlmiajanecgkibndaoabncke?hl=en-US

**Comment by mishanthrop** [comment]
Also update for Opera please, users count is the same as for Chrome (about 100).

**Comment by bartholomej** [comment]
@mishantrop Yes, I know. We are still waiting for review. In my experience sometimes it takes a week to release in opera/firerox :(
But meanwhile... Brand new extension published in Mozilla store! 
https://addons.mozilla.org/en-GB/firefox/addon/modx-queeg/

**Pull request merged**